esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -2135,3 +2135,13 @@ StringArray & getRoxieDirectAccessPlanes(StringArray & planes, StringBuffer &def
         planes.append(defaultPlane);
     return planes;
 }
+
+extern TPWRAPPER_API SecAccessFlags getDropZoneScopePermissions(IEspContext & context, const char * dropZoneName, const char * dropZoneServer, const char * dropZonePath)
+{
+    if (isEmptyString(dropZoneServer) || isEmptyString(dropZonePath))
```

Comment:
Inconsistency between the condition test || and the error message ("Both cannot be empty"). I think you need && instead

Reply:
The exception will be thrown if one or both of them are empty.